### ISSUE TITLE
install_keras(tensorflow = "gpu") not working on windows

### ISSUE THREAD

**Issue by Paul-AntoineLeboeuf** [rstudio/keras#860]
Hello everyone!

I have been trying for a long time to install this package. I am fairly sure that I did everything that the prerequisites appropriately:

- I installed CUDA 10.0 and the cuDNN version v7.6.2.24
- I installed Anaconda v1.9.7
- I installed rStudio v1.2.1335
- ran the command " install_keras(tensorflow = "gpu") " after installing keras.

As far as the prerequisites on the documentation I :
- have an Nvidia GPU (In my case gtx 1060 6gb)
- GPU drivers are at 431.60
- have CUDA 10.0 + CUPTI
- have cuDNN (>=7.4.1) ( I have 7.6.2.24)
- set to the PATH the bin and lib files of the extracted cuDNN

Thus, I expected, after running the command " install_keras(tensorflow = "gpu") ", that the command 

```
library(tensorflow)

with(tf$device("/gpu:0"), {
  const <- tf$constant(42)
})

sess <- tf$Session()
sess$run(const)
```

would tell me that everything is fine, but it is not.

Particularily, they inform me that the only device they see is 
` [ /job:localhost/replica:0/task:0/device:CPU:0 ]`

Even running the command 
`tf$python$client$device_lib$list_local_devices()`

confirms that previous statement:
 `[[1]]
name: "/device:CPU:0"
device_type: "CPU"
memory_limit: 268435456
locality {
}`

Finally, after looking at the documentation over and over again thinking I did something wrong somewhere, I'm at a loss. So, I was wondering if I had missed something during the installation.

Thank you for your help

**Comment by dfalbel** [comment]
What's the output of:

```
tf_config()
```

And what's the output of `nvidia-smi`?

**Comment by Paul-AntoineLeboeuf** [comment]
The output of ` tf_config()` is
```
TensorFlow v1.14.0 (C:\Users\PAUL-A~1\AppData\Local\Programs\Python\Python37\lib\site-packages\tensorflow\__init__.p)
Python v3.7 (C:\Users\PAUL-A~1\AppData\Local\Programs\Python\Python37\python.exe)
```

**Comment by dfalbel** [comment]
@Paul-AntoineLeboeuf you have CUDA 10.1 installed and in order for TensorFlow to work you need CUDA 10.0

**Comment by Paul-AntoineLeboeuf** [comment]
Hello @dfalbel 

As far as I know, the CUDA version described in this command only describes the maximum supported CUDA version. The following link could explain the descrepancy:

https://stackoverflow.com/a/53504578

If you take a look at the following text, you will see that CUDA 10.0 is installed.
```
C:\Users\Paul-Antoine Leboeuf>nvcc -V
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2018 NVIDIA Corporation
Built on Sat_Aug_25_21:08:04_Central_Daylight_Time_2018
Cuda compilation tools, **release 10.0, V10.0.130**
```
I will attempt to change my python version from 3.7 to 3.6 and see if anything changes with this version.

Thank you.

**Comment by dfalbel** [comment]
Hi @Paul-AntoineLeboeuf 
I am closing the issue, please reopen if you need additional help
